### PR TITLE
Fix regional weights-bug

### PR DESCRIPTION
**Pull request by pwaidelich**
This fixes the current bug that heterogenous regional damage weights for SLR and discontinuity end up being used for market and non-market damages as well, in contradiction to the PAGE-ICE excel file. To avoid that the weights are passed on between components, I named them separately following the PAGE-ICE excel where three different weights parameters are defined in the "Library Data" sheet.

As the weights for market and non-market damages are defined as Triangular(1, 1, 1) in the excel file, I did not include them into the mcs.jl file.

### CODE REVIEW

File: src/components/MarketDamages.jl [hunk 1]
```diff
@@ -37,14 +37,17 @@
     function run_timestep(p, v, d, t)
 
         for r in d.region
+            # fix the current bug which implements the regional weights from SLR and discontinuity also for market and non-market damages (where weights should be uniformly one)
+            p.wincf_weightsfactor_market[r] = 1.
```
Comment (jrising) [inline src/components/MarketDamages.jl]: This should be set in the `addmarketdamages` function, not in `run_timestep`.

File: src/components/MarketDamagesBurke.jl [hunk 2]
```diff
@@ -38,6 +38,9 @@
     function run_timestep(p, v, d, t)
 
         for r in d.region
+            # fix the current bug which implements the regional weights from SLR and discontinuity also for market and non-market damages (where weights should be uniformly one)
+            p.wincf_weightsfactor_market[r] = 1.
```
Comment (jrising) [inline src/components/MarketDamagesBurke.jl]: Same here.

File: src/components/NonMarketDamages.jl [hunk 3]
```diff
@@ -42,13 +42,16 @@
 
         for r in d.region
 
+            # fix the current bug which implements the regional weights from SLR and discontinuity also for market and non-market damages (where weights should be uniformly one)
+            p.wincf_weightsfactor_nonmarket[r] = 1.
```
Comment (jrising) [inline src/components/NonMarketDamages.jl]: Again.

**Review by jrising — CHANGES_REQUESTED** [review]
Thanks for making these changes! Can you just move the parameter-setting code to the `add...` functions?

**Comment by pwaidelich** [comment]
Like this?

**Review by jrising — APPROVED** [review]
Looks good now. Go ahead and merge it. Thanks!

**Pull request merged**